src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -406,3 +408,136 @@ Tests creating query sets on destroyed device fails.
       t.device.createQuerySet({ type, count: 4 });
     });
   });
+
+g.test('command,copyBufferToBuffer')
+  .desc(
+    `
+Tests copyBufferToBuffer command on destroyed device fails.
+  - Tests that finishing encoding fails on destroyed device
+  - Tests that submitting command fails on destroyed device
```

Comment:
How are you going to actually check this though? Lost devices don't provide error scope.

also: It may be that testing submit in each test is trivial, but if not then we can skip it. I don't think we need submit tests for command buffers containing different commands. (They're all invalid command buffers anyway.)
There's a difference between [finish, destroy, submit] and [destroy, finish, submit]. I think we only really need tests for:
- [finish, destroy, submit]
- [destroy, finish]

Reply:
SGTM. What about the error checking problem though? popErrorScope rejects on lost devices so you can't see if a validation error occurred.

Reply:
Resolving this comment for now as it is blocked by https://github.com/gpuweb/gpuweb/issues/2582. Depending on the decision there, I will re-work the tests as necessary.